### DISCUSSION TITLE
Can I add files that are non executables to phoenicis? (and multiple at once?)

### DISCUSSION THREAD

**Discussion by ghost** [PhoenicisOrg/phoenicis#2362] [Q&A]
So for example,if I download a game,then I have a folder with multiple files in it.
Then do I need to add the executable and then all the other files separately to the prefix?
Cause that would be problematic if the executable is in a sub folder,cause then it would take long to move all the files over manually to the prefix.
Then it would easier to just add the whole folder to a prefix.
So does that work?

**Comment by plata** [comment]
What do you mean by

> add files to phoenicis

Phoenicis can run executable files with an engine (e.g. Wine) via shortcuts. Which other files would you want to add and what's supposed to happen if you do so?
